src/spicy/runtime-support.cc

```diff
@@ -206,7 +206,7 @@ void rt::raise_event(const EventHandlerPtr& handler, const hilti::rt::Vector<Val
             throw InvalidValue("null value encountered after conversion");
     }
 
-    event_mgr.Enqueue(handler, std::move(vl));
+    event_mgr.Enqueue(handler, std::move(vl), util::detail::SOURCE_LOCAL, rt::current_analyzer()->GetID());
```

Comment:
If `rt::raise_event()` is called for a file analyzer or packet analyzer this will crash, I guess?